src/world/Map/TerrainMgr.cpp

```diff
@@ -405,14 +405,13 @@ void TileMap::Load(char* filename)
         return;
     }
 
-    if (header.buildMagic != 12340)  //wow version
+    if (header.buildMagic != sWorld.CacheVersion) // wow version
```

Comment:
This value should be a hardcoded value. It depends on our extractors. the TerrainMgr maybe works different on another game version....